### PR TITLE
Fixes #10928 - Default dashboard widgets for new users

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by domcleal** [comment]
Other widget tests seem to be failing because of the default widgets, but note that the assertions are the wrong way around in the tests (expected/actual numbers).

Perhaps put the after_create code into a method, then it's easy to stub from the test setup?  Or wipe the widgets again after creating the user (putting that into an FG trait may be useful.)

File: test/unit/user_test.rb [hunk 1]
```diff
@@ -842,4 +842,9 @@ def editing_self_helper
     user.timezone = ''
     assert user.valid?
   end
+
+  test 'default widgets are added on creation' do
+    assert_equal FactoryGirl.create(:user).widgets.count, Dashboard::Manager.default_widgets.count
+  end
+
```
Comment (domcleal) [inline test/unit/user_test.rb]: remove this line

**Comment by dLobatog** [comment]
@domcleal Thanks for the comments, updated - now all tests are in widget_test and the after_create call is a separate method.

**Comment by domcleal** [comment]
Merged as ad00a10 for 1.9, thanks @eLobato.

**Pull request closed without merging**